server/models/schema.sql

```diff
@@ -1849,6 +1849,17 @@ CREATE TABLE IF NOT EXISTS `voucher_item` (
   CONSTRAINT `voucher_item__voucher` FOREIGN KEY (`voucher_uuid`) REFERENCES `voucher` (`uuid`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
+DROP TABLE IF EXISTS `average_consumption_algo`;
+
+CREATE TABLE `average_consumption_algo` (
+  `id` TINYINT(3) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
It is not clear to me why we need a table for the algorithms?   They seem to be something that is strictly in the Javascript code.

Reply:
The reason we used a table is just because we had thought to keep the identifier of the algorithm to be used among the stock settings.

Reply:
That sounds fine.  Can't we use an enum for the algorithm and add a new field in the stock_settings table?  I'm still not clear why we need a new table for the algorithm info.

Reply:
In this case we could use the component **bhConstants** and modify the property
````
 `average_consumption_algo_id` TINYINT(3) UNSIGNED NOT NULL DEFAULT 3,
````
In VARCHAR with the following values 'algo1', 'algo2', 'algo' and 'algo_msh'

Reply:
I will make this modification